src/test/java/com/mfvanek/salary/calc/repositories/JpaPropertiesTest.java

```diff
@@ -0,0 +1,41 @@
+package com.mfvanek.salary.calc.repositories;
+
+import com.mfvanek.salary.calc.support.TestBase;
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.Query;
+import jakarta.persistence.QueryTimeoutException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.TestPropertySource;
+
+import static org.assertj.core.api.Assertions.assertThatNoException;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@TestPropertySource("/application-test.yml")

```

Comment:
We do not need here a custom property source. Test profile is automatically used for tests